migrations/2_deploy_contracts.js

```diff
@@ -1,5 +1,7 @@
 const Hub = artifacts.require("./Hub.sol");
+const { convert } = require('../test/helpers/math');
```

Comment:
What is `convert` exactly doing? I suggest a less generic name for the function here.

Reply:
It converts a number to 1 > 1e18, which is basically the eth > wei conversion but for circles

Reply:
I suggest renaming the function then to what it actually does for better readability of the code, or use the web3 utils methods (`toWei`)?

Reply:
Not sure I wanna call it something to do with wei, since it's not wei .. it's circles tokens and we haven't really named them yet. Only thing coming to mind is "convertToDecimals" or "convertToBaseUnit" or something like that ..

Reply:
We could call the base unit "particles" since it's the smallest indivisible part :joy: